src/index.js

```diff
@@ -648,6 +648,27 @@ const indy = {
     )
   },
 
+  async verifierVerifyProof(
+    proofRequest: IndyProofRequest,
+    proof: Indy.IndyProof,
+    schemas: Schemas,
+    credentialDefs: CredentialDefs,
+    revRegsDefs: Indy.RevRegsDefs,
```

Comment:
Hi @JamesKEbert, thanks for this update. I'm just curious, where do you get the `Indy.*` types from? I don't see any `Indy` import or definition in this file 🤔

Reply:
Ahh whoops. Those types came as a result from referencing code from a previous implementation I had done. I thought I got everything sorted but I missed that somehow. Thanks for catching that--I just pushed a commit revising the types.